scripts/plugins/extauth-hook-AD.py

```diff
@@ -164,6 +167,10 @@ def _apply_to_cache(self):
     def _format_item(self, item):
         # CA-363207: Pam cannot handle space, put it inside []
         if " " in item:
```

Comment:
I assume the substitution `item.replace(" " , "+")` can be run every time without checking for space first. It will simply do nothing if there is no space in `item`. Is `+` a legal character or are we creating a problem here by mapping space to plus and making this transformation not reversible?

Reply:
we need to check space for winbind anyway.
PBIS will reverse it back. (I can add a comment for it)